writing-docs/EXAMPLES.md

```diff
@@ -56,6 +56,15 @@ You can allow it with ok-test:
         dd 42;
     =end
 
+### Allow .perl
+
+One of the checks is to dissuade using `.perl` in tests. If needed you
```

Comment:
Probably discourage is better than dissuade, and also mention that .raku should be used instead.

Reply:
Sure ! Done :-)